### ISSUE TITLE
Can't navigate in menu by arrow on mac

### ISSUE THREAD

**Issue by rodshtein** [figma-plus/figma-plus#6]
![screencast 2019-05-07 04-00-34](https://user-images.githubusercontent.com/13717428/57244935-e9116e00-707c-11e9-90d8-ed84f172b77e.gif)

**Comment by jackiecorn** [comment]
Unfortunately the Figma Plus menu is a hack so it's very difficult to play well with the native arrow key navigation.
We will consider exploring ways to add support for the arrow keys in the future. No quick fix for now but I'll leave this issue open so we can get back to this later. Thanks!

**Comment by rodshtein** [comment]
Yeah i understand it. Search isn't work too.
May be way out is just add optional menu for FigmaPlus?

![image](https://user-images.githubusercontent.com/13717428/57294900-6f28c580-710c-11e9-86c2-b73f37c5148a.png)

**Comment by jackiecorn** [comment]
What would that button do?

**Comment by rodshtein** [comment]
Showing plugin menu? 
This is just a basic idea. Of course, two menus in a row are not the best option.

![image](https://user-images.githubusercontent.com/13717428/57295782-92ed0b00-710e-11e9-886b-ddf0e57225c7.png)

**Comment by jackiecorn** [comment]
Yeah I guess it's a little redundant to the one in the hamburger menu.
BTW did you know that most contextual commands (e.g. Advanced Rename, Fill Faces/Data) also show up in the selection right click menu? 
<img width="697" alt="Screen Shot 2019-05-07 at 4 35 23 AM" src="https://user-images.githubusercontent.com/6892625/57296318-90909f80-7081-11e9-995e-b5cab5ab826a.png">
